common.go

```diff
@@ -111,3 +115,19 @@ func SplitStringMap(m map[string]string) ([]string, []string) {
 
 	return keys, values
 }
+
+// ExecCmd executes the specified Bash script. If execution fails, the error contains
+// the combined output from stdout and stderr of the script.
+// Do not use this for scripts that produce a large volume of output.
+func ExecCmd(command string) error {
+	cmd := exec.Command("bash")
+	cmd.Stdin = strings.NewReader(command)
+
+	data, err := cmd.CombinedOutput()
+	log.Debugf("command output %v", string(data))
```

Comment:
Is this meant to always be logged, or is it leftover from local testing?

Reply:
As far as this log corresponds to `level=debug` I think it worths to be always logged.